File: ApiSuvesaPos/Negocio/Modelo/caja/movimiento_caja.vb
Namespace Modelo
    Public Class movimiento_caja
        Public Property Id As Integer
        Public Property Entrada As Boolean
        Public Property Salida As Boolean
        Public Property Monto As Decimal
        Public Property CodMoneda As Integer
        Public Property Usuario As String
        Public Property Fecha As Date
        Public Property Observaciones As String
        Public Property Anulado As Boolean
        Public Property NumApertura As Integer
        Public Property TipoCambio As Decimal
    End Class
End Namespace

File: ApiSuvesaPos/ApiSuvesaPos/Modelo/caja/movimiento_caja.cs
using System;

namespace Modelo
{
    public class movimiento_caja
    {
        public int Id { get; set; }
        public bool Entrada { get; set; }
        public bool Salida { get; set; }
        public decimal Monto { get; set; }
        public int CodMoneda { get; set; }
        public string Usuario { get; set; }
        public DateTime Fecha { get; set; }
        public string Observaciones { get; set; }
        public bool Anulado { get; set; }
        public int NumApertura { get; set; }
        public decimal TipoCambio { get; set; }
    }
}
